contributor-guides/glossary.md

```diff
@@ -7,3 +7,518 @@ Astro components are the basic building blocks of any Astro project. They are HT
 ## Islands
 
 Islands is a frontend architecture that results in better frontend performance by helping you avoid monolithic JavaScript patterns and stripping all non-essential JavaScript from the page automatically. The technique that this architectural pattern builds on is also known as partial or selective hydration. In Astro, an “island” refers to any interactive UI component on the page. Read more about islands in the [Astro Islands](https://docs.astro.build/en/concepts/islands/) page.
+
+-----
+
+SARAH RESOLVING CONFLICTS
+Additions from this PR are added below the existing content
+
+-----
+
+## Adaptors
+
+Adaptors are essential for deploying Astro projects to different hosting platforms. Each adaptor is tailored to a specific host, such as Netlify, Vercel, or Cloudflare Pages, ensuring compatibility and streamlined deployment processes. Adaptors handle the nuances of each platform, providing a seamless integration for Astro projects.
+
+For more details, see:
+- [Astro Adaptor API Reference](https://docs.astro.build/en/reference/adapter-reference/)
+ 
+## API
```

Comment:
Not a change, but just a note that we will need to classify these based on Astro-relatedness at some point in the future.